### DISCUSSION TITLE
[Feature] Automatic separation into albums upon load

### DISCUSSION THREAD

**Discussion by Jamminroot** [immich-app/immich#3733] [Feature Request]
### The feature

Hey, amazing application and I really appreciate your work.

TL;DR: I'd like to be able to divide my uploads into folders and\or automatically assign albums to them

I've got a bunch of various folders on my phone: camera roll, some photos imported from GoPro, some photos in a separate folder based on the occasion (like "Vacation" or "Holiday Trip" etc.), bunch of memes to show my wife, screenshots, pictures from my work and others.
I'd like all of them to be backed up and be accessible, but the problem is that all of them get mixed into one flat collection, thus causing all them to be show in the same collection.

Now with any new onboarding to using the application user would get thousands of assets, all mixed into one collection. Assigning albums manually would be a problem - for both already uploaded assets, or every time one downloads a new funny gif\picture.

Also worth mentioning that when I download tv series, memes, work images or anything non-related to actual photos I take, It'd be great to be able to ignore all those irrelevant assets in face recognition: that would allow to reduce the resource cost on the feature, and "Face roll" would not be swarmed with faces of random ppl \ images (imagine having a face of a dear one somewhere in the end of the list, after a ton of irrelevant persons :) )

The only alternative I can have atm is to ignore everything but actual camera roll.

Hope that makes sense, and I really hope that it would be considered, as it is quite a painful downside for such a great tool.

Also I apologise if that is a duplicate, but I didn't really find a good description here on github or reddit.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1678

**Reply by Jamminroot** [reply]
Not entirely, but yes, thanks for pointing out. Ill cross-link with #1678 and close this discussion
